asapdiscovery-dataviz/asapdiscovery/dataviz/viz_targets.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+
+
+# enum for allowed targets
+# TODO make configurable from YAML file
+class VizTargets(Enum):
```

Comment:
I have made the possible visualisation targets enumerated in a central location here.